### PR TITLE
(SUP-4138) Restore missing PCP Connections panel

### PR DESCRIPTION
**Pull request by m0dular**

### CODE REVIEW

File: files/Orchestrator_performance.json [hunk 1]
```diff
@@ -50,8 +51,6 @@
             "mode": "palette-classic"
           },
           "custom": {
-            "axisCenteredZero": false,
-            "axisColorMode": "text",
```
Comment (bastelfreak) [inline files/Orchestrator_performance.json:54]: what's the purpose of all those axis attributes and why are they gone?

Comment (m0dular) [reply files/Orchestrator_performance.json:54]: I'm not sure why small changes like this show up in the export.  All I did was make sure I had the latest dashboard from `main` on my box and add the new panel.  Maybe it's a Grafana version thing?

**Comment by bastelfreak** [comment]
@m0dular I see the  "PCP Connections" panel, but it doesn't show data in my test environment. I will dig around a bit.

**Comment by m0dular** [comment]
Does it show anything if you run a test from the cli?

```
export INFLUX_TOKEN=<token>
telegraf --test --debug --config /etc/telegraf/telegraf.conf --config /etc/telegraf/telegraf.d/pcp_metrics.conf
```

There should be a measurement for `mbean=default:name\=puppetlabs.pcp.connections` in the output.

**Comment by bastelfreak** [comment]
Today I tested it in an environment with just a primary. telegraf outputs the metrics but grafana doesn't display anyhting. It is possible that the panel only displays data from compilers, not from primaries? I will test that tomorrow.

**Comment by m0dular** [comment]
I did some testing as well, and for some reason the field is named `puppetlabs.pcp.connections` and not `default:name\=puppetlabs.pcp.connections`.  I have no idea why that is, because the name returned by the metrics api is `default:name=puppetlabs.pcp.connections`, which is also what shows up in the manual `telegraf` test.  I _swear_ this worked for me before, but in my latest tests the field is indeed named `puppetlabs.pcp.connections` in Influx, so that's what we need in the Grafana panel. I'll push an update.

**Comment by bastelfreak** [comment]
I just confirmed this works in my test environment with a single primary,thanks alot! will test in another env with compilers as well.

**Comment by bastelfreak** [comment]
@m0dular works fine for me even in bigger setups. Can you merge and release it?

**Comment by m0dular** [comment]
@bastelfreak or @MartyEwings could you approve the PR for posterity?

**Review by bastelfreak — APPROVED** [review]

**Comment by bastelfreak** [comment]
ah sure, sorry that I forgot it!

**Pull request merged**